cfgov/v1/util/ref.py

```diff
@@ -262,6 +262,11 @@ def is_blog(page):
         return True
 
 
+def is_event(page):
```

Comment:
This file is doing a lot, and the way categories, slugs, and page types interact is pretty confusing. I didn't think this task should include any refactoring, but at some point it would be nice to simplify it, if possible.

Reply:
Yeah — refactoring that file comes up every time we have to touch it, but we never do ☹️ .